diff --git a/companion/lib/Graphics/Controller.js b/companion/lib/Graphics/Controller.js
index 9ab24f46..2f0ab96d 100644
--- a/companion/lib/Graphics/Controller.js
+++ b/companion/lib/Graphics/Controller.js
@@ -220,7 +220,7 @@ class GraphicsController extends CoreBase {
 			'Companion-symbols4': generateFontUrl('NotoMusic-Regular.ttf'),
 			'Companion-symbols5': generateFontUrl('NotoSansLinearA-Regular.ttf'),
 			'Companion-symbols6': generateFontUrl('NotoSansLinearB-Regular.ttf'),
-			// 'Companion-emoji': generateFontUrl('NotoColorEmoji-compat.ttf'),
+			'Companion-emoji': generateFontUrl('NotoColorEmoji-compat.ttf'),
 			'5x7': generateFontUrl('pf_tempesta_seven.ttf'),
 		})
 		this.fonts = FontLibrary.families
